tfsdklog/sink.go

```diff
@@ -30,13 +31,22 @@ const (
 	// standard error. Setting this environment variable to another file
 	// path will write logs there instead during tests.
 	envLogFile = "TF_LOG_PATH"
-)
 
-var sink hclog.Logger
+	// envAccLogFile is the environment variable that controls where log
+	// output from the provider under test and the Terraform binary (and
+	// other providers) will be written during tests. Setting this
+	// environment variable to a file will combine all log output in that
+	// file. If both this environment variable and TF_LOG_PATH are set,
+	// this environment variable will take precedence.
+	envAccLogFile = "TF_ACC_LOG_PATH"
 
-func init() {
-	sink = newSink()
-}
+	// envLogPathMask is the environment variable that controls per-test
+	// logging output. It should be set to a fmt-compatible string, where a
+	// single %s will be replaced with the test name, and the log output
+	// for that test (and only that test) will be written to that file.
+	// Setting this environment variable will override TF_LOG_PATH.
+	envLogPathMask = "TF_LOG_PATH_MASK"
```

Comment:
Maybe this is a good time to mention that when set, only includes the provider under test?

```suggestion
	Only the logs for the provider under test are included.
	envLogPathMask = "TF_LOG_PATH_MASK"
```